src/functionalTest/resources/scenarios/RIA-7338-review-sign-language-appellant-flag.json

```diff
@@ -0,0 +1,157 @@
+{
+  "description": "RIA-7338 review sign language for appellant with legal rep",
+  "enabled": "true",
```

Comment:
Should this test be enabled by $featureFlag.isSubmitHearingRequirementsEnabled ?